staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -302,7 +302,7 @@ func IsHTTPHeaderName(value string) []string {
 	return nil
 }
 
-const envVarNameFmt = "[-._a-zA-Z][-._a-zA-Z0-9]*"
+const envVarNameFmt = "[-:._a-zA-Z][-:._a-zA-Z0-9]*"
 const envVarNameFmtErrMsg string = "a valid environment variable name must consist of alphabetic characters, digits, '_', '-', or '.', and must not start with a digit"
```

Comment:
can we also change the message to include the fact that we now support : ?

Reply:
Good catch! Updated.